tests/cloudformation/checks/resource/aws/example_SecurityGroupRuleDescription/SecurityGroupRuleDescription-FAILED-4.json

```diff
@@ -1,40 +0,0 @@
-{
```

Comment:
@twmartin Why delete this test?

Reply:
The new `SecurityGroupRuleDescription-FAILED-3.json` now plays the role of the "json-egress-rule-with-no-description" test.

Reply:
I believe the 3 failed/passed test examples remaining in this PR provide sufficient coverage for the `SecurityGroupRuleDescription` Cloudformation check:
- SG resource w/ inline rule - `SecurityGroupRuleDescription-(FAILED|PASSED).yaml`
- Ingress rule resource - `SecurityGroupRuleDescription-(FAILED|PASSED)-2.yaml`
- Egress rule resource - `SecurityGroupRuleDescription-(FAILED|PASSED)-3.yaml`

For that reason, I deleted the examples above number 3.